messages.pot

```diff
@@ -941,12 +827,21 @@ msgstr ""
 msgid "try_it"
 msgstr ""
 
+msgid "print"
```

Comment:
Do we need a new key here? We already have the keyword print traslated in content/keywords?

Reply:
We could even fetch print hello from the first adventure, then we don't have to translate anything exta:

https://github.com/hedyorg/hedy/blob/411a9b3e9da3f8749b9c4982bb3a59b1c12ccb66/content/adventures/en.yaml#L18

Reply:
We could, but this would make the code a bit more complex as we have to split on the whitespace as well for styling. New key might not be ideal, but I think the most straight-forward solution.